docs/src/wiki-deprecated/ports.md

```diff
@@ -30,21 +30,21 @@ provide powerful wiring constructs described later.
 
 (Chisel 3.2+)
 
-Chisel 3.2+ introduces an API `DataMirror.modulePorts` which can be used to inspect the IOs of any Chisel module, including MultiIOModules, RawModules, and BlackBoxes.
+Chisel 3.2+ introduces an API `DataMirror.modulePorts` which can be used to inspect the IOs of any Chisel module, including Modules, RawModules, and BlackBoxes.
```

Comment:
I am a little confused by this sentence. does "any Chisel module" mean "Any Chisel.Module, chisel3.Module, and ..."? If not can you clarify.

Reply:
Yes, it means anything that inherits from `BaseModule`, I'll clarify